styles.css

```
@@ -0,0 +1,106 @@
+* {
+  box-sizing: border-box;
```

Andrea suggested me yesterday to add also 
```suggestion
  box-sizing: border-box;
  margin: 0;
```